docs/tutorial.rst

```diff
@@ -374,12 +374,6 @@ and query the data, this time from node B.
    ssh -l ha-admin -t `vm_ip monitor` \
      pg_autoctl perform switchover --pgdata monitor
 
-   # watch the progress
-   ssh -t -l ha-admin `vm_ip monitor` -- \
-     watch -n 1 -d pg_autoctl show state --pgdata monitor
-
-   # press CTRL-C to stop watching
```

Comment:
Why remove this section?

Reply:
Because it's not needed anymore, now that the `pg_autoctl perform failover` shows what's happening thanks to using the notifications from the monitor.